projects/plugins/jetpack/tests/php/core-api/wpcom-endpoints/test-class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -418,12 +418,12 @@ public function menu_item_url_data() {
 	 *
 	 * @throws \ReflectionException Noop.
 	 * @dataProvider menu_item_update_data
-	 * @covers ::parse_update_count
+	 * @covers ::parse_markup_data
 	 */
-	public function test_parse_update_count( $menu_item, $expected ) {
+	public function test_parse_markup_data( $menu_item, $expected ) {
```

Comment:
Do we want to provide another set of test data here to cover the non-"count" path?

You could use the `@ dataProvider ` to pass multiple data sets to the same test.

https://blog.martinhujer.cz/how-to-use-data-providers-in-phpunit/